src/drivers/drivers.jl

```diff
@@ -85,9 +86,24 @@ function set_driver!(p::Properties, ::POSIX)
 end
 
 function __init__()
-    DRIVERS[API.h5fd_core_init()] = Core
-    DRIVERS[API.h5fd_sec2_init()] = POSIX
-    @require MPI="da04e1cc-30fd-572f-bb4f-1f8673147195" include("mpio.jl")
+    # disable file locking as that can cause problems with mmap'ing
+    if !haskey(ENV, "HDF5_USE_FILE_LOCKING")
+        ENV["HDF5_USE_FILE_LOCKING"] = "FALSE"
+    end
+
+    fapl = HDF5.init!(HDF5.FileAccessProperties())
+    API.h5p_set_fapl_core(fapl, 4096, false)
```

Comment:
Nowhere in particular (other than typical Linux page size).  TBH, I suspect this could be set to 0 here because it's only used to setup fapl, which in turn is only used to get the ID of the Core driver.  This fapl is never used to actually create anything.  FWIW, I think this value defaults to 8192 elsewhere in HDF5.jl, but it's not clear where that value comes from.